src/navsat_transform_node.cpp

```diff
@@ -37,7 +37,9 @@ int main(int argc, char ** argv)
 {
   rclcpp::init(argc, argv);
 
-  robot_localization::NavSatTransform trans;
+  auto node_ = rclcpp::Node::make_shared("navsat_transform_node");
```

Comment:
Same comment re: non-member variables. Do the new standards suggest that this should have a trailing underscore?

Reply:
No, I think it was just a copy pasta error

Reply:
Thanks, will fix it.